### PR TITLE
Fleury Algo in C++ added

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes PR #13

### CODE REVIEW

**Comment by Riyazul555** [comment]
@iuliagroza please check this PR

File: Fleury/Fleury.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// This code is contributed by Riyazul555
```
Comment (iuliagroza) [inline Fleury/Fleury.cpp:1]: See #74.

File: Fleury/Fleury.cpp
Comment (iuliagroza) [inline Fleury/Fleury.cpp:1]: Rename to "Fleury/fleury.cpp".

**Comment by Riyazul555** [comment]
Done @iuliagroza

File: Fleury/Fleury.cpp
Comment (iuliagroza) [reply Fleury/Fleury.cpp:1]: Please rename before approving the merge

**Comment by Riyazul555** [comment]
Screenshot (701)

Its already renamed @iuliagroza 
You merge this pR I will fix it in next PR

**Comment by Riyazul555** [comment]
Done @iuliagroza

**Review by iuliagroza — APPROVED** [review]

**Pull request merged**